### DISCUSSION TITLE
✨ Welcome: Introduce Yourself Here ✨

### DISCUSSION THREAD

**Discussion by aza** [earthspecies/project#22] [General]
Hello everyone, and welcome to the ESP Discussions intro thread. Please introduce yourself 🤖

**Comment by ShaunSpinelli** [comment]
Howzit, my name is Shaun. Currently working as an ML developer working mainly in computer vision but keen to start working/learning  in some other areas and this seem like an exciting and challenging opportunity to get stuck in.

**Reply by aza** [reply]
Hi @ShaunSpinelli! Very excited to welcome you to the project! @radekosmulski can give you a sense of what good first projects might be for the Earth Species Library!

**Reply by radekosmulski** [reply]
Hi @ShaunSpinelli! Pleasure to meet you! 🙂

We are actively growing [the ESP library](https://github.com/earthspecies/library). There are already several projects there which contain introductory notebooks, walking one through how to get started with bioacoustic datasets. All the data is publicly hosted and freely available. I wonder if it might not be a good place to get started? Showing some way of working with the data, finding something interesting about the datasets, could be of great value - this is a resource frequented by bioacustic researchers and conservationists so maybe through the examples that we share they would have an easier time applying the methods do their datasets (or maybe sharing their data with us so that we can give them a hand 🙂). More challenging datasets coming soon.

Let us know please how does that sound to you or whether you are looking for something else. Maybe we will be able to help one way or another.

**Reply by bs** [reply]
👋

**Comment by sainisatish** [comment]
Hi all my name's satish saini , Currently I'm pursuing B.Tech from Moradabad Institute of technology.I want to be a ML developer .

**Reply by aza** [reply]
Welcome, Satish!

**Reply by radekosmulski** [reply]
Pleasure to meet you @sainisatish! 👋

**Reply by bs** [reply]
Welcome! 🐋

**Comment by JoseMMontoro** [comment]
Hi! My name is Jose Montoro. I'm an old friend of @bs from our days of going to yoga to the same place in SF! Hi Britt, it's been a while! 👋

It was through Britt that I first found out about ESP, and I found it fascinating. I love the mission and the approach. I'm currently a Language Engineer at Apple with a background in Linguistics. I'm very interested in all the new approaches to understanding language that technology provides. What ESP is trying to accomplish is both an incredibly stimulating intellectual exercise and a very powerful take on probably the most pressing issue of our time.

That is to say, I'd love to contribute! I read in the comments above that the ESP library might be a good place to start. Is that still the case? Is there anything in particular that would help the project right now? Maybe a general exploration/visualization/summarization of part the ESP library data would still be useful? 

Nice to meet you all!

**Reply by radekosmulski** [reply]
Hi @JoseMMontoro! Great pleasure to e-meet you 🙂

> Maybe a general exploration/visualization/summarization of part the ESP library data would still be useful?

That would be beyond lovely! The idea is that the library could also work as a source of inspiration for way in which researchers can work with their data. The techniques that you mention would certainly lie at the core of that! 

There is also a related discussion, on the [audio-embeddings project here](https://github.com/earthspecies/audio-embeddings/issues/1#issuecomment-699946134). 

Hearing from you is great and any contributions will be extremely appreciated. Please let me know if there is anything I could help you with or should you have any ideas you would like to discuss. 

We currently have 3 datasets in the ESP library and are in the process of working on adding more.

**Reply by bs** [reply]
> Hi! My name is Jose Montoro. I'm an old friend of @bs from our days of going to yoga to the same place in SF! Hi Britt, it's been a while! 👋

Hey @JoseMMontoro! How are you!?

> a Language Engineer at Apple with a background in Linguistics. I'm very interested in all the new approaches to understanding language that technology provides. 
> 
> That is to say, I'd love to contribute! I read in the comments above that the ESP library might be a good place to start. Is that still the case?

Your insight as a linguist would be supremely helpful. Do you have thoughts on how your experience might be best applied? Do you know other linguists that would be interested in contributing to the conversation? (We're working with [Kevin Ryan](http://www.people.fas.harvard.edu/~kevinryan/) on sperm whales.)

As mentioned in [this thread](https://github.com/earthspecies/audio-embeddings/issues/1#issuecomment-699946134), we're presently focused on two lines of research:

- [unsupervised audio-audio](https://github.com/earthspecies/audio-embeddings)
- [cocktail party problem](https://github.com/earthspecies/cocktail-party-problem)

And indeed, we are building out the library. Any help you want to provide here would be well received! For example, we're working with [Yossi Yovel](http://www.yossiyovel.com/index.php/publications)'s [Egyptian fruit bats](https://github.com/earthspecies/project/discussions/24). A good first issue might be duplicating some of the classification notebooks in /library on this.

So glad you're here!

**Reply by JoseMMontoro** [reply]
Thank you both! Let me digest all the info you presented and I'll come back with potential next steps that I can help with.

Excited to be here!

**Comment by chandojo** [comment]
Hello! My name is Celena and I discovered this project from the "Two Heartbeats A Minute" episode on Invisibilia.  I'm a newb to comp sci but sociolinguistics has been a hobby of mine.  This project is fascinating and makes me think of Dr. Dolittle (if anyone remembers that movie)! I'm a little overwhelmed and intimidated about the knowledge and skills behind a project like this but I hope there is a way I can contribute.  If anything, I can atleast learn from you all!

**Reply by kzacarian** [reply]
Welcome @chandojo !

**Reply by radekosmulski** [reply]
Pleasure to meet you Celena! 👋 🙂

Ditto on Dr. Dolittle! That was the first reference that came to my mind when I was trying to explain what this undertaking is about to my kids 😄

**Comment by glicerico** [comment]
Hi all! 
The aforementioned Invisibilia episode is also what brought me to this project, which I found quite inspiring from a philosophical, scientific and environmental perspective.
I'm an astronomer-converted-to-AI researcher interested in Artificial General Intelligence (AGI). My focus has been on unsupervised grammar learning, and so many of the tools mentioned in the [roadmap](https://github.com/earthspecies/project/blob/main/roadmaps/ai.md) are related to my work (although I haven't worked on translation before). 
It's been a few months that I wanted to check your efforts in more detail, and it seems I will finally have some more time in my hands to do it. I read the recommendations above, so I will start by exploring the mentioned repos :)

**Reply by radekosmulski** [reply]
Hi Andres! 🙂 Super happy to see that you found your way to our github, great to see you here

**Comment by ryjombari** [comment]
Hello Friends.  I am a biological oceanographer studying marine mammal communication and human noise impacts in the ocean.  Glad and grateful to connect.

**Reply by aza** [reply]
So honored to have you here @ryjombari!

**Reply by ryjombari** [reply]
Thanks Aza!  Some wonderful communication happening in Monterey Bay right now (live stream at https://www.mbari.org/soundscape-listening-room/)

**Reply by bs** [reply]
🐋 Now this is the stream I wanted to tune into this morning :)

So glad to have you hear, @ryjombari. Appreciate you and all your work.

**Comment by crywolfproject** [comment]
Hi and, more important than my introduction, thank you for what you are doing! I sat through the recent webinar and was humbled by the expertise but excited to see that the application of your research is approachable to citizen scientists like myself. My interest in this effort comes from three aspects of my background. First, a PhD in systemic functional linguistics (a very non-Chomskian approach to language modeling) in the 90's prior to ML/AI being used in that field. Second, a 25 year career in enterprise software tech and more lately in hardware suppliers of various sensors/MCUs used in the IoT market. Third, 50 years of living in the Greater Yellowstone ecosystem, and within the past 10 years studying animal behavior as a citizen-scientist. I am not a computer scientist with the fabulous skills of Aza and others, but I understand how to bring scalable products to market within the tech world. Several of my counterparts and I have used those relationships to launch AI/ML hardware projects that bring together researchers and computer scientists building models that can used to study animal communication. An example of this is https://www.hackster.io/contests/ElephantEdge -- pairing cutting edge smart collars (with audio input) with AI models across various species. My company is also working with Microsoft as they launch their new project Santa Cruz hardware for vision/audio AI on the edge use cases. Currently, we are using this research in the Greater Yellowstone to create models of cattle behavior that can predict grizzly and wolf predation events so as to help deter such events that result in euthanization. We are also deploying AI Vision Edge "trail cameras" with long battery life to record and signal unique animal behavior and reduce false positives for researchers. I really look forward to taking whatever I learn from the Earth Species science and helping bridge the gap to research being done around Yellowstone National Park to bring further awareness of the multi-modal communication used by species throughout this unique ecosystem. In particular, there is a veritable army of "citizens" who go through Yellowstone Park with recording devices on their cell phones to their commercial video platforms who could be lobbied to contribute vocalizations they capture while vacationing in the area. Even better, there is a more focused army of locals (many with Masters and PhDs in relevant scientific fields) who could be utilized to get more targeted vocalizations (with video often) via a coordinated effort...after all, they live here to see this native ecosystem at work and are out gathering data or helping formal researchers daily. I've already shared your work with several of them in Gardiner MT and we're discussing which species we might focus on first.

**Reply by radekosmulski** [reply]
Super excited to e-meet you Jeff! 🙂

**Reply by aza** [reply]
It is such a pleasure to meet you Jeff! The work you are doing looks incredible and Elephant Edge is so cool. CC @kzacarian

**Reply by crywolfproject** [reply]
Have you seen this paper: https://evlab.mit.edu/assets/papers/Grand_et_al_2022_NHB.pdf

**Comment by FBDV** [comment]
Hi,
I need help in my PoC project.
I am not an IoT expert, so this is a MVP, the cheaper the better. Hardware must be able to handle vibrations. I will put it inside a 3D print plastic box to protect it.
I need to record sounds from 8 microphones, simultaneously, using an IoT hardware and store it in a micro SD card or transmit to cloud (via a used smartphone). Sounds must be in WAV format (not compress or alter in anyway). Microphones need to have a 4 meters long cables, each, so impedance must be in mind.
Them I need a software to show sound patterns on the notebook screen.
Can anyone help me?
Cheers,

**Comment by shivam-131** [comment]
Hi Everyone, I am Shivam Gupta, a junior undergrad at IIT Kanpur. I am exploring research in deep learning and am right now working around some brain inspired algorithms. I also have very recently started volunteering for several data science for social good nonprofits. At my university I also lead a club which provides machine learning solutions to non-profits, all pro-bono. I have a little backgroud in NLP but I am always keen to learn and explore new research areas. I would like to contribute to this project.
I loved the problem statement you guys are working on, they seem very unique and fresh. I am excited!

**Reply by radekosmulski** [reply]
hello 👋 great to see here Shivam! 🙂

**Reply by shivam-131** [reply]
Hi Radek. It would be great if you could guide me about how I should proceed further. :)

**Reply by radekosmulski** [reply]
The two projects we are working on right now are the [cocktail party problem](https://github.com/earthspecies/cocktail-party-problem) and [unsupervised audio translation](https://github.com/earthspecies/unsupervised-audio-translation). We also continue to flesh out [the ESP library](https://github.com/earthspecies/library). For instance, this [entire section](https://github.com/earthspecies/library/tree/main/egyptian_fruit_bat) was contributed by a member of our community.

Might be the ESP library could be a good place to start. A potential project could be to get started with one of the datasets that we make available and demonstrate a machine learning or deep learning technique applied to that data. We hope that the library can serve both as a source of data useful for research and also as a repository of practices of working with bioacoustic data that others can leverage in their work. 

An alternative path would be to seek out an open source dataset and contribute it to our library. We have a document that is still in the works outlining [the steps](https://github.com/earthspecies/library/blob/main/howto_add_a_new_dataset_to_the_library.md) to do so.

**Comment by samjoy** [comment]
Hi, My name is Sam Joy. I am a data scientist doing work in the field of NLP and deep learning. I have just started my AI journey professionally after completing my graduation with a Masters in Computer Science. I got to know of this project after having listened to the podcast episode "Two Heartbeats A Minute". I must say this project is truly inspiring and impactful. I am hope to provide any contributions that I can to this wonderful project.

**Reply by radekosmulski** [reply]
Great to meet you Sam! 👋

**Reply by samjoy** [reply]
Hi Radek, can you tell how I should proceed?

**Reply by radekosmulski** [reply]
Hi Sam! I give an overview of what we are currently working on [here](https://github.com/earthspecies/project/discussions/22#discussioncomment-136362). We do nearly all our work in the public and everything should be available on our github profile 🙂

**Reply by samjoy** [reply]
Thank you radek. I will have a look at it.

**Comment by doctorbmac** [comment]
Hello! 
My name is Brad, I recently completed my undergrad in CS and am currently a graduate student working towards my MS in Computational Linguistics. This project is incredible, and inspiring as I work on my own research utilizing word embedding. I am fascinated by semantics and would love to contribute, are “the cocktail party problem” or the ESP library still good places to start?

Thanks in advance and best of luck!

**Reply by radekosmulski** [reply]
Hi Brad! Great to meet you :slightly_smiling_face: I believe everything that is in the open is a fair game to contribute to :slightly_smiling_face: I think the ESP library might offer a gentler way to get started. I am also not sure if we have pushed our latest and greatest work on the CCP to github yet.

If you have any questions at any point let me know! :slightly_smiling_face:

**Comment by asross** [comment]
Hey! My name is [Andrew](https://asross.github.io/), and I'm a nearly-graduated PhD student in machine learning at Harvard.

Although this isn't really my area of expertise, I wanted to share a [slightly crazy idea](https://docs.google.com/document/d/18Wpv7idS1khLDRH3CDSE2aZNy5PziGvz6Oyqtc2CZrQ/edit?usp=sharing) I have for collecting a small amount (or potentially even a large amount) of _supervised_ animal vocalization data, i.e. vocalization data that we know pertains to specific, well-defined concepts. My sense is that having that kind of data could really help even if the bulk of the work is done in an unsupervised setting. For example, if you had a set of vocalizations for the concept of "large" and a set of vocalizations for the concept of "small," you might be able to also label a _direction_ in embedding space, which you could then re-use to help analyze (differences between) embeddings of unknown meaning.

Would be curious to hear what you think!

**Reply by radekosmulski** [reply]
Hi Andrew, a pleasure to meet :wave: :slightly_smiling_face:!

I absolutely agree - being able to infer something about the embedding space based on a couple of supervised examples would be super interesting and very, very valuable. Enjoyed reading your doc with the experiment design - shared all of this with our team internally :slightly_smiling_face:

**Comment by crywolfproject** [comment]
One of the big remaining questions in animal linguistics/communication theory is “do they have enough semantic domains to actually create directions in an embedded space that can be mapped to human languages”. In my opinion, it’s the 60T question. If we just start with birds, where the genetics of the FOXP2 gene seems to suggest a common evolution to basic vocalizations, and we just start with what we know about three common functions of bird calls: alarm calls, attraction calls (songs), and contact calls…AND WE DECIDE BETWEEN A WORD AND A SYLABLE, then can you brilliant ML folks create an embedded space that maps to a known human language and create a translation tool. Researchers know enough about a black-capped chickadee alarm call to tease out which part of the “sentence” are “words” that indicate the type of species that is threatening them (e.g. pigmy owl vs great horned owl), but we think this is due to a repetition of syllables rather than a different spectrogram. In other words, the number of calls indicates the meaning in the case of a species threat, not the sound of the call. That said, the “dee” portion of a chick-a-dee three or four note/word alarm “sentence” definitely seems to be associated with the type of threat. And perhaps with your embedded space we can figure out what the rest of the words in that same sentence mean (e.g. location of threat?).

Or, perhaps better than starting with birds, start with Con Slobodovich’s research on prairie dogs (which has even identified dialects between species that could be used to ground truth your embedded spaces), which is sophisticated enough to create an embedded space which they might be able to help those same researchers figure out some of the other vocalizations a prairied dog makes that we have no clue on.

Anyway, great to have you aboard. I still believe this project is the best hope we have at creating a grammar of Chickadees…or Prairie Dogs…or…

**Reply by radekosmulski** [reply]
Thank you Jeff - great to hear from you, thank you for sharing your thoughts :slightly_smiling_face:

**Comment by rshirale** [comment]
Hello, I am Rahul, Nice to be here. Very excited about the mission. I am an Engineer with more than 12 years of experience, moving from IT to Machine learning.

**Reply by radekosmulski** [reply]
Hi Rahul, pleasure to meet :slightly_smiling_face:

**Comment by patwater** [comment]
Hi I'm a computational social scientist, and stumbled on this project listening to a podcast on a road trip on an annual rafting trip with some old science-ey friends. 

Out of curiosity, are you all thinking about playing various sounds back to their species to try to better understand what they mean through observed behavior? The umwelt issues in this inspiring initiative will be quite tricky to tackle! 

Also by the way this link to the [cocktail party problem](https://github.com/earthspecies/cocktail-party-problem) in the Project Readme is broken :/

Cheers, 

PA

**Reply by radekosmulski** [reply]
Hi Patrick, great to meet you :wave: :slightly_smiling_face: 

Replay experiments are definitely something we consider. Completely agree with you on the observation regarding the umwelt issue :slightly_smiling_face: Since we don't have access to ground truth, replay experiments might prove very useful :slightly_smiling_face: 

If you enjoy podcasts, [here](https://www.wsj.com/podcasts/wsj-the-future-of-everything/how-to-talk-to-animals-and-know-what-theyre-saying-back/4002fe99-f796-4fde-9dd5-ed71a01337c3) is another one where @aza talks about one way this technology could be used for good (spoiler alert: we could use it to broadcast to animals information on impending danger). 

Thank you very much for your message! And thank you for the heads up on the broken link - we have made significant progress on the CPP and are in the process of reorganizing the materials. At the latest, within the next couple of weeks, we plan to do a complete update and push it all out to the repo :slightly_smiling_face:

**Reply by patwater** [reply]
How would you go about doing the replay experiments? I wonder if there's a creative way to use UAVs dressed up to look like a human or something or maybe that'd just weird out the whale. Having a little bit of empathy it'd be pretty weird to have a sound emitter dropped from the sky shouting random words that may or may not make sense.

Also I was playing around a bit with the orca dataset and one thing that strikes me is that we humans hear 0-17khz ish and the orcas optimal hearing frequency seems to be around 20khz and go up to 125khz. Just wondering how you all think about developing an intuition or exploring the data as a step towards building a model given that context. I suppose you could do something like false color with infrared or ultravioled spectra? 

Thanks for sharing the podcast btw!

**Reply by patwater** [reply]
Do you have any recs on scientific papers looking into what ppl consider signal in an orca call? Like do we have a sense of what in this type of spectrogram signfies the "orca call"? 
![image](https://user-images.githubusercontent.com/1587811/114258932-1bb9f480-997f-11eb-92a8-820e21fb8e6e.png)

**Reply by radekosmulski** [reply]
You are 100% spot on with regards to the empathy bit :slightly_smiling_face: I do not know what the answer is - before any replay experiments will be done, a lot of thought would need to be given to the considerations that you mention. Also, an experiment design would need to be done, we would need to have some understanding of what we are looking for and of course one would need to monitor the well-being of the animals! I completely agree with you that carrying out such an experiment could be a major source of stress. For us, this is still something that is down the road, we are not considering any such experiments right now. Should we get to this point we would be embedded in a team of biologists and collaborators, they would have a much better understanding of everything that goes into preparing such an event than I do :slightly_smiling_face: 

You are spot on with regards to the audible frequency range. It is not easy for us to get a feel for the data. Also, we have to be very careful to introduce as little anthropocentric bias into our work as we can. For instance, for working with sounds mel-spectrograms are often used. They work well when analyzing human speech or for sound event detection, but the mel-scale has been created experimentally through studying how we hear! This can lead to a very obvious source of bias but there are many other assumptions that we make along the way, ones that can be more subtle but potentially very damaging. Not an easy nut to crack :slightly_smiling_face: 

One way of getting a feel for the data is replaying it with a smaller sample rate. The sound is distorted but it gives us an option to experience the data we work with.

With regards to analyzing the spectrogram, I am not sure there is an easy answer. An orca call would be most likely defined as any sound originated by the animal that we can pick up on! The best way to go about identifying it in the spectrogram is to listen to a bunch of audio and cross-reference the visualization. I have not worked with orca sounds but I would imagine that - assuming their vocalizations are clicks - those are the pronounced vertical lines both in the waveform and on the spectrogram.

The way this plays out is that the more spectrograms you see, the better you become at recognizing sounds in them :slightly_smiling_face: The way I understand this is that researchers spend literally hundreds if not thousands of hours pouring over the data they collect in something like Raven and they develop a very strong intuition what a sound might look like. I have observed them look at a spectrogram and identify the bird species or speak to what was happening just by scrolling through a spec ("oh, here it started to rain" etc). Quite impressive :slightly_smiling_face:

**Reply by patwater** [reply]
@radekosmulski thank you for the very thoughtful and in depth response! I caught this Smithsonian lecture with a NOAA chief scientist yesterday on my bike ride which reminded me of your comments about how to structure a replay experiment. Tbh the way they went about blasting sonar that's been known to cause whales to strand and then seeing what happens was somewhat surprising in the lack of empathy though I suppose that really shouldn't be surprising given that's really the status quo right now. Anyway some fascinating science and useful reference point that you might enjoy if it's not already on your radar. See here: https://podcasts.apple.com/us/podcast/ocean-lectures/id467098851?i=1000097438332

**Comment by wpercybrooks** [comment]
Hi! My name is Winston, I'm currently a college professor at a university in Barranquilla, Colombia. As a background I have a PhD in Electrical and Computer Eng. during which I got first exposed to ML/DL ideas while working on voice conversion. Since then I have done applied research and teaching in several ML subjects (NLP, CV, SmartGrid protections, Audio analysis/synthesis). 

I learned about ESP recently thanks to a post on some random Internet forum. Reading about your goals brought me back to my days as a senior undergraduate student, when I was involved in an attempt to use then current automatic speech recognition techniques (pre-Deep learning era) to analyze the vocalizations of the Cotton Top Tamarin monkey, an endangered endemic species to my neck of the woods. I really wish that project had gotten continuity in time to be able to reach its original goals, and now that I found about you guys I'm really excited about the opportunity to participate in some capacity on something even more ambitious.

I've been lurking a little around this board and your whole set of repositories, some really interesting ideas already there. Personally I agree with  @reedflyfarm in that we should focus first not in translation per se, but on how machine learning can help us to determine the structure (semantics/sintaxis) of the communication of particular species. Once that structure can be reasonably extracted (ultimately going the unsupervised way, but also exploiting some supervised data when available) the we can worry about how to translate it into some human intelligible format (and then the reverse direction: human -> non-human animal). The work on unsupervised segmentation of human speech is seems particularly relevant to me, if we can duplicate/improve those approaches on different human languages then we could be relatively confident on their potential to identify meaningful audio segments for other species. Of course I realize that there are many other challenges and research avenues to explore related to the ultimate objective of ESP.

Going back to my Cotton Top Tamarin monkeys (BTW check them out in Google they are seriously cute!) Do you guys know about 'Proyecto Titi'? (https://www.proyectotiti.com/en-us/) They are a U.S.-backed conservation initiative to save the Tamarins in their endemic habitat (close to where I live). They have been operating for a long time, and among their conservation efforts they do research related to the monkeys, including their vocalizations (https://www.proyectotiti.com/en-us/Cotton-top-Tamarins/Publications). I understand the Tamarins have been found to have a fairly complex and well understood set of vocalizations, plus the people at proyecto Titi record many wild and captive individuals. It would be worth reaching out to them in order to grow the ESP library with another interesting study case. 

Well, this first post is getting too long, so I'll finish here! Thanks if you read the whole thing! Looking forward to participating in the ESP initiative.

Cheers!

**Comment by muskedunder** [comment]
Hi! My name is Colin. I'm a machine learning engineer working with audio models that run on tiny battery powered devices (microcontrollers) for a startup here in Sweden. I recently participated in two birdcall recognition competitions on Kaggle, which opened my eyes to bioacoustics.

I found out about ESP from @radekosmulski on Twitter. I find the project absolutely fascinating and can't wait to dive deeper when I get the time!

**Comment by cparmet** [comment]
Hey everybody! I'm a data scientist in nonprofit healthcare and communication, and a buddy of animals. I've just discovered your magnificent project. I'm stoked to dig in and learn and hope to contribute!

PS: I don't know enough about the ESP methodology & toolbox yet to know if this is relevant, so I'm sorry if it proves off topic. But I wanted to share a study that made a splash in the medical literature today: **Neuroprosthesis for Decoding Speech in a Paralyzed Person with Anarthria** ([study](https://www.nejm.org/doi/full/10.1056/NEJMoa2027540), and a nice lay summary in [STAT News](https://www.statnews.com/2021/07/14/machine-learning-translate-brain-signals-text)). They used DL models, a language model, and a Viterbi decoder to detect words and then predict sentences in real time from the brain signals of a man who'd lost the physical ability to speak. Only N=1 and not bridging species, but caught my eye while thinking about ESP. (Aside from the prospect of relieving a lot of human suffering, if this line of research pans out. Which stirs my heart, man.)

**Comment by pearlhyacinth** [comment]
Hello. My name is Melody and I am currently an art practice PhD in virtual reality and zoosemiotics. It is wonderful to discover your project. I am specifically interested in the umwelt and worlding of animals and am trying to creatively visualize the voices of animals in creative social virtual worlds and also using VR tools to make sculptures inspired by different animal languages. Mine is really a creative approach - more to open up thinking about meaning in non-human voices. But I am writing about current research in this field - like yours. Thanks for your work. So exciting. This is my site: https://melodyowen.net

**Comment by leuc** [comment]
Hi all,

i am a random hacker, with 20+ years of work experience in IT, who became fascinated by the diversity and complexity of beluga whale  vocalizations.

I stumbled over [Watkins marine mammal sound database](https://whoicf2.whoi.edu/science/B/whalesounds/) and have been working to make that dataset _more_ accessible for researchers. (Just a personal open source side project, i am not associated with any institution.)

The projects goals are outlined on the website: https://marine-mammal.soundwave.cl/about.html

Watkins database has some **good potential** to be used in the research of non-human languages. If you guys have any ideas, applications or tasks, i would be happy to help with the technical side of things.

**Comment by janichols31** [comment]
Hi, 
My name is James. I currently work in the wind industry but am developing an interest in the use of machine learning techniques to represent the workings of the brain. I was searching to find people who wanted to develop human-animal communication for the purpose of encouraging greater conservation efforts - and here you are. I also believe that this research could enable higher-fidelity between human individuals leading to greater empathy.
Reading the website, it seems as if the research is very much focused on analysis of the auditory output of communication and I wondered whether the project would consider working with electro-encephalogram or other neural activity data.

**Comment by gradjitta** [comment]
Hi, I am Aditya and I currently work as a ML engineer in a Startup in Finland. The goal of Earth Species Project truly resonates with me and would like to contribute to any open source projects you are undertaking. What are the projects you folks working on currently? 
Feel free to reach me at:  aditya (dot) jitta (at) gmail (dot) com

**Comment by AnnaKGarr** [comment]
Hello! My name is Anna and I have a background in animal cognition and have consulted for AI modeling research projects. I also have a Ph.D. in neuroscience and decision-making. I would love to join your company and have just submitted an application. Thank you for your time and attention to my application and for doing such great work in this field and for this world!

**Comment by keoughkath** [comment]
Hi all, 

I'm Kathleen, a computational biologist by training with expertise in comparative and functional genomics currently working to translate evolutionary innovations into human therapeutics. I'm very intrigued and inspired by the mission of ESP and keen to get involved. Currently planning to explore the github to see which projects are most active and how I might contribute. 

Cheers!
Kathleen

**Comment by juliencatanese** [comment]
Hello everyone,

I'm Julien, researcher and data scientist with a PhD in neuroscience. I love and have substantial experience in animal cognition and have written scientific papers in this field. 

ESP's mission to decode the language of animals captivates me, and I'm eager to contribute my expertise and channel my energy into leading this incredible journey to resounding success.

**Comment by GvanR88** [comment]
Hi everyone, 

I am a field guide in South Africa and I interpret animal behavior for international guests on Safari in Africa. I am passionate about nature and find the communication systems between plants animals and humans fascinating. 

I'm looking forward to learning more from you all in this thread.

**Comment by MuriaRoberts** [comment]
Hi everyone! Great to be part of the Earth Species Project Discord chat. I have been researching and experimenting with multispecies education for over two decades now. It is so exciting to see developments in technology that hold the power to deepen our understanding of non-human species - and enable us to access their perspectives. These multispecies perspectives will be critical in guiding our efforts to solve global sustainability challenges. It’s a shared planet. Looking forward to exchanging ideas with you all and learning together as part of this amazing project!
Kindly always, Muria

**Comment by bobduru** [comment]
Hello everyone! I'm Emmanuel Fernandez, currently collaborating with the LISSE lab in Quebec to study beluga whale vocalizations using deep learning. My research focuses on detecting and classifying these calls with CNNs and a fine-tuned AVES model. As a Data Science and AI graduate with a focus on NLP, I find your research into understanding the meaning behind these vocalizations absolutely fascinating. If there's any way I can contribute to your projects, whether big or small, I'd love to help out. Just let me know!

**Comment by xolotl** [comment]
Hi folks! I may have met you in the Discord or community calls. I'm interested in ESP because I believe intraspecies understanding can help humans navigate our insufficient commitment to planetary coexistence. As for me: I connect people, ideas, and technologies to make the world better, most recently working as the Director of Communications & Community at Creative Commons and now independently at [Nudgital](https://nudgital.com/). I’ve worked across a wide variety of public and private institutions, focusing on community development, digital communications, meaningful education, open technologies, and sustainable growth. I live in Portland, Oregon USA among a lot of crows and with some other cats and humans. Learn more about me on [my blog](https://xolotl.org).

**Comment by MuriaRoberts** [comment]
Hi Nate!
Welcome and nice to meet you.
Kindly,
Muria Roberts
Founder/Director, Multispecies Education International MEI

**Comment by anahitalebourdiec** [comment]
Hi all!
Super excited to join the community! I am a vet and neuroscientist focusing my research on animal consciousness. I leverage AI-IoT to hack sounds, rhythms, odours (semiochemical signalling) and postures/facial expressions in pets, livestock and wild animals. I currently work as Head of Alternative Innovation at Ceva Animal Health. I also work with plant electrophysiology/ML to translate plants needs and reduce water consumption (and phytochemicals use, etc...) through precision farming (www.vegetalsignals.com) I advocate knowledge based economy and ecology. I am thrilled to join the game!

**Comment by estreeter49** [comment]
Hello to you all!
My name is Elizabeth and I am so excited that the recent developments in AI have brought us to a place where we can advance the cracking of communication codes within, and possibly between, various species! My professional background has been in communications, and my personal life very much entwined with ecology, animal behavior, neuroscience, and consciousness. As I read through these posts and the interesting things people have been called to study and contribute to, I send virtual kudos and thanks to you all for advancing these important areas of work! So happy to be here and to more closely follow along!
